CoreGraphFramework/src/au/gov/asd/tac/constellation/graph/StoreGraph.java

```diff
@@ -2136,7 +2136,7 @@ private boolean evaluateLayerQuery(final GraphElementType elementType, final int
         final Boolean result = QueryEvaluator.evaluatePostfix(expression);
 //        System.out.println("Result calculated: " + result);
 
-        return result == null ? false : result;
+        return result;
```

Comment:
Actually, instead of returning result, just return `QueryEvaluator.evaluatePostfix(expression)` and remove the need for the result variable